cost-analyzer/templates/kubecost-metrics-service-template.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.kubecostMetrics }}
+{{- if .Values.kubecostMetrics.exporter }}
+{{- if .Values.kubecostMetrics.exporter.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kubecost.kubeMetricsName" . }}
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: {{ (quote .Values.kubecostMetrics.exporter.port) | default (quote 9005) }}
+spec:
+  ports:
+  - name: tcp-metrics
+    port: {{ .Values.kubecostMetrics.exporter.port | default 9005 }}
+    protocol: TCP
+    targetPort: {{ .Values.kubecostMetrics.exporter.port | default 9005 }}
```

Comment:
Is there a better way of defining the constant defaults like 9005 that we use here?

Reply:
Not really, but maybe there are ways to clean it up a bit.